### PR TITLE
Use AAPT2 to dump badging instead of AAPT

### PR DESCRIPTION
**Pull request by jmatsu**
Related to #117 

We should not use AAPT anymore. The format of AAPT2 output is almost same to one of AAPT so we can migrate to AAPT2 easily.

### CODE REVIEW

File: spec/android_apk_spec.rb [hunk 1]
```diff
@@ -39,7 +39,6 @@
         {
           filepath: fixture_file("invalid", "corrupt_manifest.apk"),
           error: AndroidApk::UnacceptableApkError,
-          error_message: /AndroidManifest\.xml is corrupt/
```
Comment (jmatsu) [inline spec/android_apk_spec.rb:42]: This message won't be included for AAPT2.

File: lib/android_apk.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "fileutils"
+require "forwardable"
```
Comment (jmatsu) [inline lib/android_apk.rb:4]: To delegate property accesses to AAPT2's result class.

**Review by enomoto-kazuya — APPROVED** [review]
LGTM!

**Comment by jmatsu** [comment]
Thank you~

**Pull request merged**